src/hooks/use-sesjon-status.tsx

```diff
@@ -38,9 +38,12 @@ export const useSesjonStatus = (): { sesjonStatus: SesjonStatus } => {
 	useEffect(() => {
 		let timeout: number | undefined;
 
-		if (sekunderTilSesjonUtloper !== null) {
+		if (sekunderTilSesjonUtloper !== null && sekunderTilSesjonUtloper !== undefined) {
 			const msTilSesjonUtloper = sekunderTilSesjonUtloper * 1000;
 
+			if (timeout) {
+				window.clearTimeout(timeout);
```

Comment:
Make sure timeout is cleared if is exists before assinging varible to a new timeout